.github/workflows/release.yaml

```diff
@@ -200,6 +200,48 @@ jobs:
       env:
         GH_TOKEN: ${{ secrets.REPO_ACCESS_TOKEN }}
 
+  build_push_docker_image:
+    needs: [build, release]
+    name: Build and push Docker image
+    permissions:
+      contents: read
+      packages: write # To push the Docker image to ghcr.io
```

Comment:
Because `./.github/workflows/_build_docker.yaml` is also called from an untrusted context for testing PRs, we need to refactor the Docker build workflow and push the image either as a step here in the `release.yaml` or as a separate RWF. We want to tighten the permissions and access to pull requests.

Reply:
Fixed in https://github.com/oracle-samples/macaron/pull/76/commits/24da909332996f30775f68257a844b0d2b9cbb6a